sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/ErrorParserSuite.scala

```diff
@@ -39,6 +39,23 @@ class ErrorParserSuite extends AnalysisTest {
       context = ExpectedContext(fragment = "order by q\ncluster by q", start = 16, stop = 38))
   }
 
+  test("Illegal characters in unquoted identifier") {
+    // scalastyle:off
```

Comment:
Why do you need this?

Reply:
Scalastyle complains about non-ASCII characters.

Reply:
which one? seems all chars are ASCII.